aemedge/blocks/table/table.css

```diff
@@ -56,3 +78,108 @@
 .table.bordered table td {
   border: 1px solid;
 }
+
+
+/* new code */
+
+*{
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+.schedule.table, .playoff.table {
+  width: 100%;
+  overflow-x: auto;
+}
+
+.schedule.table table .lightgrey {
+  background-color: var(--light-grey-color);
+}
+
+.schedule.table table .single-column-row{
+  background-color: var(--dark-color);
+  width: 100%;
+  text-align: center;
+}
+
+.playoff.table table tbody tr td:not(:nth-child(4)) {
+  background-color: var(--light-grey-color);
+}
+
+.schedule.table .single-column-row.lightgrey td{
+  text-align: left;
+}
+
+.playoff.table table .single-column-row.lightgrey td{
+  background-color: var(--dark-color);
+}
+
+
+
+.schedule.table p, dl, ol, ul, pre, blockquote {
+margin-top: 2px;
+margin-bottom: 5px;
+}
+
+
+
+
+.schedule {
+  display: flex;
+  justify-content: center;
+  width: 100%;
+  font-size: 16px;
+}
+
+
+
+div.content-details-wrapper div.default-content-wrapper ,div.section.table-container div.default-content-wrapper{
+  text-align: center;
+}
+
+@media (width >= 600px) {
+  .schedule.table table {
+    font-size: var(--body-font-size-xs);
```

Comment:
This font size is already declared at line #6, BUT I think this is wrong anyway as the table on the live site has the same font size of 16px for all sizes.